playbooks/roles/crafty-bot/files/docker-compose.yml

```diff
@@ -32,5 +32,5 @@ services:
     <<: *django
     image: registry.gitlab.com/opencraft/dev/crafty-bot:flower-latest
     ports:
-      - "5555:5555"
+      - "127.0.0.1:5555:5555"
```

Comment:
@gabor-boros, this is going to make Flower inaccessible from outside of the instance, no? We might either open it or add an Nginx config to do a proxy for this.

Reply:
I'm planning to add an NGINX location for it.

Reply:
We should probably add support for [url_prefix|https://flower.readthedocs.io/en/latest/config.html#url-prefix] in [here](https://gitlab.com/opencraft/dev/crafty-bot/-/blob/master/compose/production/django/celery/flower/start#L10) and introduce a new variable to specify it. Currently [https://crafty.opencraft.com/celery-flower/] has broken statics and links.